UnleashClient/strategies/Strategy.py

```diff
@@ -66,6 +88,12 @@ def parsed_constraints(self) -> Iterator[Constraint]:
             for constraint in segment["constraints"]:
                 yield Constraint(constraint_dict=constraint)
 
+    @property
+    def parsed_variants(self) -> Variants:
+        return Variants(
+            variants_list=self.variants, group_id=self.parameters.get("group_id", "")
```

Comment:
Not sure default stickiness is right here - It is the feature_name when called from Feature.  Would a random string fit better here?

Reply:
Oh good point actually!

Hmm, now that I think about it, can group_id actually even be missing? Since this goes through the new flow, this should always be set, right? Maybe we don't need the default value at all